src/main/java/org/broadinstitute/consent/http/db/UserDAO.java

```diff
@@ -175,4 +175,12 @@ void updateUser(@Bind("displayName") String displayName,
             @BindList("datasetIds") List<Integer> datasetIds,
             @BindList("roleNames") List<String> roleNames);
 
+    @RegisterBeanMapper(value = User.class)
+    @SqlQuery("SELECT u.* FROM dacuser u "
+      + " LEFT JOIN user_role ur ON ur.user_id = u.dacuserid "
+      + " LEFT JOIN roles r ON r.roleid = ur.role_id "
+      + " WHERE r.name = 'SigningOfficial' "
```

Comment:
nit: Prefer lower case string comparisons:
```suggestion
      + " WHERE LOWER(r.name) = 'signingofficial' "
```